Ochibobo/CountZeros.java

```diff
@@ -0,0 +1,34 @@
+package com.goclinics.examples;
+/*
+    Java program to count zeros in an Array
+
+    @author (Warren Ochibobo)
+
+*/
+class CountZeros{
+    /*
+    *
+    * Count the number of zeros in an array
+    *
+    *
+    * @param arr - Integer array from which zeros are to be counted
+    * @param n - size of the arr
+    *
+    * @return - count of zeros in the array
+    */
+    public int countZeros(int [] arr, int n){
```

Comment:
You can get the length of the array using inbuilt fn size or len

Reply:
Sure thing @lakivisi. I was banking on the method that passes the array to pass its size to, that is, the function to get the size of the array will be called by the caller method. But I understand what you mean.